coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1020,7 +1020,23 @@ func (s *server) FailJob(ctx context.Context, failJob *proto.FailedJob) (*proto.
 			return nil, err
 		}
 
-		s.notifyWorkspaceBuildFailed(ctx, workspace, build)
+		// Only notify auto build failures
```

Comment:
I would rather that this comment explain what classifies as an auto build (i.e. not initiated by a user)